src/styles/index.scss

```diff
@@ -1,3 +1,129 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
+  width: 200px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+
+  background-position: center;
+  background-size: cover;
+  margin: 33 20 0;
+}
+
+.card__title {
+  display: block;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40 17 0;
```

Comment:
The margin shorthand property in `.card__title` is missing units for its values, similar to the `.card__img` class. It should be `margin: 40px 17px 0;`.